setup.py

```diff
@@ -1,9 +1,16 @@
 import os.path
 import subprocess
-from distutils.core import setup
 
 from suricata.update.version import version
 
+try:
+    from setuptools import setup
+    USE_SETUPTOOLS = True
+except ImportError:
+    from distutils.core import setup
+    USE_SETUPTOOLS = False
+
+
```

Comment:
minor: extra line space.. Or is this style, 2 spaces after import block?

Reply:
Yes. We'll get E302 if this is not done. http://pycodestyle.pycqa.org/en/latest/intro.html